tests/test_internal_structures.py

```diff
@@ -82,7 +84,9 @@ def test_evaluator_detector_naming(mitigation_outputs: Tuple[List[str], List[str
     COMPLYING_OUTPUTS, REFUSAL_OUTPUTS = mitigation_outputs
 
     d = MitigationBypass()
-    attempt = garak.attempt.Attempt(prompt="testing prompt", lang=d.lang_spec)
+    attempt = garak.attempt.Attempt(
+        prompt=garak.attempt.Message(text="testing prompt"), lang=d.lang_spec
```

Comment:
While I don't think we actually have any detectors with multiple lang values in `lang_spec` this should probably be explicit.
```suggestion
        prompt=garak.attempt.Message(text="testing prompt"), lang=d.lang_spec.split(",")[0]
```

Reply:
OK, so plugin `.lang_spec` is a spec and `lang` (in e.g. `Message`) is a single choice. Which is clear from var name. Let's aim to continue with this explicit & clear convention.